spec/requests/api/v1/users/account/profiles_spec.rb

```diff
@@ -6,9 +6,10 @@
   let!(:account) { create(:account, :with_user) }
   let(:headers) { { Authorization: auth_header(account) } }
   let(:params) { {} }
+  let(:profile_url) { '/api/v1/users/account/profile' }
```

Comment:
Do we need to check not just GET actions. Could we have a situation, something like

```
Post.all.each do |post|
  post.category.update(name: 'test')
end
```